gnosis/test/verification.k

```diff
@@ -15,10 +15,12 @@ module VERIFICATION
  // -------------------------------------
     rule #infinity +Int _ => #infinity
     rule #infinity -Int _ => #infinity
-    rule #infinity >=Int _ => true
-    rule _ <=Int #infinity => true
-    rule #infinity <Int _ => false
+    rule A <=Int #infinity => true
+    	requires #getKLabelString(A) =/=String "#infinity_VERIFICATION"
+    rule #infinity <Int A => false
+			requires #getKLabelString(A) =/=String "#infinity_VERIFICATION"
 
+		rule minInt(#infinity, #infinity) => #infinity
```

Comment:
@denis-bogdanas this is bothering.  is it because you use tab instead of space?

Reply:
fixed formatting in last forced push.